utils/coffea/job_submission_helper.py

```diff
@@ -57,6 +57,30 @@ def __get_client(executor_name, n_workers, cores, memory, disk, time, partition,
             job_script_prologue=job_script_prologue,
         )
 
+    elif "ETP" in executor_name:
```

Comment:
What about using lowercase everywhere, as we do for `lpccondor`? It's true it's a string, so anything could be written inside without following PEP recommendations! But just to make naming consistent.